src/guake/terminal.py

```diff
@@ -74,8 +74,8 @@
      r"^[-a-zA-Z0-9\/\_\.\ ]+\.?[a-zA-Z0-9]+\:[0-9]+",
      r"^(.*)\:([0-9]+)"),
     ("line containing '/home/' absolute path",
-     r"(/home/[a-zA-Z0-9_\-\./]+)(:[0-9]+)?",
-     r"(/home/[a-zA-Z0-9_\-\./]+):?([0-9]+)?")
+     r"(/home/[-a-zA-Z0-9_\./]+)(:[0-9]+)?",
```

Comment:
Also, quick open won't work with filenames with backslashes.
if you try to quick open this file:
`$ echo "text" > 'some\file.txt'`
`$ echo 'some\file.txt:1'`
command will be
`DEBUG    Opening file some\file.txt at line 1`
where \f if is an escaped char.